website/docs/blueprints/data-analytics/binpacking-custom-scheduler-eks.md

```diff
@@ -0,0 +1,56 @@
+---
+sidebar_position: 6
+sidebar_label: Binpacking for Amazon EKS
+---
+
+
+# Binpacking for Amazon EKS
+
+## Introduction
+By default, the [scheduling-plugin](https://kubernetes.io/docs/reference/scheduling/config/#scheduling-plugins) NodeResourcesFit use the ```LeastAllocated``` for score strategies. For the long running workloads, that is good because of high availability. But for batch jobs, like Spark workloads, this would have high cost. By changing the from ```LeastAllocated``` to ```MostAllocated```, it avoids spreading pods across all running nodes, leading to higher resource utilization and better cost efficiency. In this post, we will show you how to enable binpacking with Amazon EKS.
+
+
+## Deploying the Solution
+Please follow [this link](https://github.com/aws-samples/custom-scheduler-eks) to deploy the custom scheduler.
```

Comment:
Can you add the deployment step directly here and ask the users to refer to the aws-samples project for more details. 
Also, please explain more details of any caution that users needs to be aware while deploying this scheduler

Reply:
There is a section called considerations, but indeed we should give the adoption guidance for the users